### PR TITLE
 [refactor](planner) In the runtime filter, use crc_hash replace original murmur_hash

### PR DESCRIPTION
**Pull request by Mryange**
# Proposed changes

After testing, crchash is 4 to 5 times faster than murmurhash.

## Problem summary

When be_exec_version is 1, the original murmurhash method is still used. If updated, the new crchash method will be used.

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

File: be/src/exprs/runtime_filter.cpp [hunk 1]
```diff
@@ -1011,6 +1017,11 @@ class RuntimePredicateWrapper {
 
     // When _column_return_type is invalid, _use_batch will be always false.
     bool _use_batch;
+
+
+    // When _use_new_hash is set to true, use the new hash method. 
+    // This is only to be used if the be_exec_version may be less than 2. If updated, please delete it.
+    bool _use_new_hash;
```
Comment (HappenLee) [inline be/src/exprs/runtime_filter.cpp:1024]: const bool

File: be/src/vec/exprs/vbloom_predicate.cpp [hunk 2]
```diff
@@ -67,10 +67,21 @@ Status VBloomPredicate::execute(VExprContext* context, Block* block, int* result
...
+
+        // When _be_exec_version is equal to or greater than 2, we use the new hash method. 
+        // This is only to be used if the be_exec_version may be less than 2. If updated, please delete it.
+        if(_be_exec_version  >= 2){
+            for (size_t i = 0; i < sz; i++) {
+                auto ele = argument_column->get_data_at(i);
+                const StringRef v(ele.data, ele.size);
+                ptr[i] = _filter->find_new_hash(reinterpret_cast<const void*>(&v));
+            }
+        }else {
```
Comment (HappenLee) [inline be/src/vec/exprs/vbloom_predicate.cpp:79]: code format

Comment (HappenLee) [inline be/src/vec/exprs/vbloom_predicate.cpp:79]: code format

**Comment by Mryange** [comment]
run buildall

File: be/src/vec/exprs/vbloom_predicate.cpp [hunk 3]
```diff
@@ -67,10 +67,20 @@ Status VBloomPredicate::execute(VExprContext* context, Block* block, int* result
     auto ptr = ((ColumnVector<UInt8>*)res_data_column.get())->get_data().data();
     auto type = WhichDataType(remove_nullable(block->get_by_position(arguments[0]).type));
     if (type.is_string_or_fixed_string()) {
```
Comment (BiteTheDDDDt) [inline be/src/vec/exprs/vbloom_predicate.cpp:69]: Why the new hash only used at string type?

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 33.54 seconds
 stream load tsv:          450 seconds loaded 74807831229 Bytes, about 158 MB/s
 stream load json:         23 seconds loaded 2358488459 Bytes, about 97 MB/s
 stream load orc:          72 seconds loaded 1101869774 Bytes, about 14 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230404090728_clickbench_pr_124614.html

**Comment by Mryange** [comment]
run buildall

**Pull request closed without merging**